code/include/playfab/PlayFabApiSettings.h

```diff
@@ -15,13 +15,13 @@ namespace PlayFab
 #else
     private: // But, static library memory size and alloc issues mean it always needs to exist
 #endif
-        std::string developerSecretKey; // Developer secret key. These keys can be used in development environments.
+        String developerSecretKey; // Developer secret key. These keys can be used in development environments.
 
```

Comment:
is this file a public include?  if so, it can't use mem hooked types.  if not, then ignore comment

Reply:
its not.  Next PR reorganizes project files in a more obvious way

---
In reply to: [604531508](https://github.com/jplafonta/XPlatCSdk/pull/1#discussion_r604531508) [](ancestors = 604531508)